tritium-metrics-jvm/src/main/java/com/palantir/tritium/metrics/jvm/JvmMetrics.java

```diff
@@ -190,6 +183,16 @@ protected Ratio getRatio() {
         });
     }
 
+    private static long totalHeapPlusNonHeap(MemoryMXBean memoryBean, ToLongFunction<MemoryUsage> longFunction) {
+        MemoryUsage heap = memoryBean.getHeapMemoryUsage();
+        MemoryUsage nonHeap = memoryBean.getNonHeapMemoryUsage();
```

Comment:
I don't think these are allowed to be null by the api contract (though it's possible for an implementation to return null)